cli/pkg/kctrl/cmd/package/builder/create.go

```diff
@@ -0,0 +1,450 @@
+	pkgFileLocation := create.PkgLocation + "/pkgBuilder.yml"
+	if err != nil {
+		return err
+	}
+	err = writeToFile(pkgFileLocation, packageData)
+	if err != nil {
+		create.Ui.ErrorLinef("Unable to create package file. %s", err.Error())
+		return err
+	}
+	return nil
+}
+
+func (create CreateStep) createAndPrintPackageCR() error {
+	str := `# Great, we have all the data needed to builder the package.yml and packageMetadata.yml. 
+# This is how the package.yml will look like
+# 	$ cat package.yml
+`
+	create.Ui.PrintBlock([]byte(str))
+	//TODO we should start getting the data from pkgBuilder rather than create
+	pkg := create.populatePkgFromCreate()
+
+	//TODO: remove this comment. Marshal will make yaml/json
+	jsonPackageData, err := json.Marshal(&pkg)
+	if err != nil {
+		return err
+	}
+	yaml.JSONToYAML(jsonPackageData)
+	packageData, err := yaml.JSONToYAML(jsonPackageData)
+	if err != nil {
+		return err
+	}
+	pkgFileLocation := create.PkgLocation + "/package.yml"
+	if err != nil {
+		return err
+	}
+	err = writeToFile(pkgFileLocation, packageData)
+	if err != nil {
+		create.Ui.ErrorLinef("Unable to create package file. %s", err.Error())
+		return err
+	}
+
+	output, err := util.Execute("cat", []string{pkgFileLocation})
+	if err != nil {
+		return err
+	}
+	create.Ui.PrintBlock([]byte(output))
+	return nil
+}
+
+func (create CreateStep) createAndPrintPackageMetadataCR() error {
+	str := `
+# This is how the packageMetadata.yml will look like
+# 	$ cat packageMetadata.yml
+`
+	create.Ui.PrintBlock([]byte(str))
+	//TODO we should start getting the data from pkgBuilder rather than create
+	pkgMetadata := create.populatePkgMetadataFromCreate()
+	jsonPackageMetadataData, err := json.Marshal(&pkgMetadata)
+	if err != nil {
+		return err
+	}
+	packageMetadataData, err := yaml.JSONToYAML(jsonPackageMetadataData)
+	if err != nil {
+		return err
+	}
+	pkgMetadataFileLocation := create.PkgLocation + "/packageMetadata.yml"
+	if err != nil {
+		return err
+	}
+	err = writeToFile(pkgMetadataFileLocation, packageMetadataData)
+	if err != nil {
+		create.Ui.ErrorLinef("Unable to create package metadata file. %s", err.Error())
+		return err
+	}
+
+	output, err := util.Execute("cat", []string{pkgMetadataFileLocation})
+	if err != nil {
+		return err
+	}
+
+	create.Ui.PrintBlock([]byte(output))
+	return nil
+}
+
+func writeToFile(path string, data []byte) error {
+	err := os.WriteFile(path, data, 0644)
+	if err != nil {
+		return err
+	}
+	return nil
+}
+
+func (create CreateStep) populatePkgMetadataFromCreate() v1alpha1.PackageMetadata {
+	packageMetadataContent := v1alpha1.PackageMetadata{
+		TypeMeta:   v1.TypeMeta{Kind: "PackageMetadata", APIVersion: "data.packaging.carvel.dev/v1alpha1"},
+		ObjectMeta: v1.ObjectMeta{Name: create.FqName},
+		Spec: v1alpha1.PackageMetadataSpec{
+			DisplayName:      strings.Split(create.FqName, ".")[0],
+			LongDescription:  "A long description",
+			ShortDescription: "A short description",
+			ProviderName:     "VMWare",
```

Comment:
dont think we want VMware to be provider of packages.

Reply:
Adopted

Reply:
Leaving it empty.